docker-import.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/sh
```

Comment:
Might want to clarify in a comment or documentation if this script is intended to be used by humans, or if it is intended to be consumed as an entrypoint script. 



```
** No /Library/Application Support/AnkerMake/AnkerMake_64bit_fp/login.json credentials detected **
** No /Library/Application Support/Ankermake/AnkerMake_64bit_fp/login.json credentials detected **
** No //Users/spencer.owen/.wine/drive_c/users/spencer.owen/AppData/Local/AnkerMake/AnkerMake_64bit_fp/login.json credentials detected **
** No //Users/spencer.owen/.wine/drive_c/users/spencer.owen/AppData/Local/Ankermake/AnkerMake_64bit_fp/login.json credentials detected **
** Importing /Users/spencer.owen/Library/Application Support/AnkerMake/AnkerMake_64bit_fp/login.json credentials **
[*] Loading cache..
[*] Initializing API..
[*] Requesting profile data..
[!] Config import failed: API request failed: 401 Unauthorized (auth token might be expired: make sure Ankermake Slicer can connect, then try again)
[*] ServiceManager: Shutdown complete
Configuration import failed :(
** Importing /Users/spencer.owen/Library/Application Support/Ankermake/AnkerMake_64bit_fp/login.json credentials **
[*] Loading cache..
[*] Initializing API..
[*] Requesting profile data..
[!] Config import failed: API request failed: 401 Unauthorized (auth token might be expired: make sure Ankermake Slicer can connect, then try again)
[*] ServiceManager: Shutdown complete
Configuration import failed :(
** No /Users/spencer.owen//Users/spencer.owen/.wine/drive_c/users/spencer.owen/AppData/Local/AnkerMake/AnkerMake_64bit_fp/login.json credentials detected **
** No /Users/spencer.owen//Users/spencer.owen/.wine/drive_c/users/spencer.owen/AppData/Local/Ankermake/AnkerMake_64bit_fp/login.json credentials detected **
```

Are users expected to run these steps (this is what I had to do to get it working)
1. `docker compose up`
2. run script
3. `docker compose restart`

Reply:
Ah, hm, good point.

Through the web ui, we can click the "reload services" button, but yeah, I'll incorporate a `docker restart` on the container at the end of the script.